content/en/events/2020/kcc2020/faq.md

```diff
@@ -0,0 +1,211 @@
+  - [Are friends and family allowed?](#are-friends-and-family-allowed)
+  - [Where is the celebration being held?](#where-is-the-celebration-being-held)
+  - [When will I get my invite?](#when-will-i-get-my-invite)
+  - [What sort of activities are planned?](#what-sort-of-activities-are-planned)
+- [Discord Information](#discord-information)
+  - [How do I join?](#how-do-i-join)
+  - [Why do I need to verify myself once I connect (complete captcha)?](#why-do-i-need-to-verify-myself-once-i-connect-complete-captcha)
+  - [Why do I need to agree to abide by the Code of Conduct?](#why-do-i-need-to-agree-to-abide-by-the-code-of-conduct)
+  - [Why can I only see the instructions-to-connect and info channels?](#why-can-i-only-see-the-instructions-to-connect-and-info-channels)
+  - [What is this YAGPDB.xyz and why is it messaging me?](#what-is-this-yagpdbxyz-and-why-is-it-messaging-me)
+  - [How do I display my pronouns in Discord?](#how-do-i-display-my-pronouns-in-discord)
+  - [What are the `#<channel>-info` channels for?](#what-are-the-channel-info-channels-for)
+  - [What are the `#<channel>-links` channels for?](#what-are-the-channel-links-channels-for)
+  - [How do I know if someone is a Moderator, Staff Member or an Admin?](#how-do-i-know-if-someone-is-a-moderator-staff-member-or-an-admin)
+  - [Someone is spamming/harassing me, what should I do?](#someone-is-spammingharassing-me-what-should-i-do)
+  - [Discord is running slowly or stuttering, what should I do?](#discord-is-running-slowly-or-stuttering-what-should-i-do)
+  - [Help! I have a question?](#help-i-have-a-question)
+
+## General Information
+
+### What is the Kubernetes Contributor Celebration?
+
+The Kubernetes Contributor Celebration is an attempt to reclaim that and
+celebrate our accomplishments. It's a time for us to relax, chat and do something
+fun with your fellow contributors!
+
+### When is it?
+
+It will be from the afternoon (Pacific Time) December 10th through the weekend,
+ending on December 13th.
+
+### How do I register?
+
+Please fill out the [registration form].
+
+[registration form]: https://forms.gle/51tqQgxuHxLaeU1P8
+
+### Are friends and family allowed?
+
+Allowed and encouraged! Please have them register as a friend or family member
+and use the email you registered with as the reference. They must agree to abide
+by the Code of Conduct while attending.
+
+
+### Where is the celebration being held?
+
+The celebration will be hosted on [Discord], a hosted online communication
+platform similar to Slack, but with a focus on voice and video in addition to
+text base chat. It’s been built to support the hosting of a
+[safe, positive and inclusive space]. 
+
+
+### When will I get my invite?
+
+Discord server invites will be sent out closer to the event.
+
+
+### What sort of activities are planned?
+
+There will be trivia and other games available. For more information see the
+[activities] page.
+
+[activities]: /events/kcc2020/activities/
```

Comment:
every link needs to be updated to reflect the new file paths.
Find -> replace
`/events/kcc2020` -> `/events/2020/kcc`